### PR TITLE
Documentation :  add powerdns oid integration

### PR DESCRIPTION
**Pull request by Kitof**

### CODE REVIEW

File: docs/content/en/integration/openid-connect/powerdns/index.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+draft: false
+images: []
+menu:
+  integration:
+    parent: "openid-connect"
+weight: 620
+toc: true
+community: true
+aliases:
+  - /docs/community/oidc-integrations/powerdns.html
```
Comment (james-d-elliott) [inline docs/content/en/integration/openid-connect/powerdns/index.md:15]:
```suggestion
```

File: docs/content/en/integration/openid-connect/powerdns/index.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+          - https://powerdns.example.com/oidc/authorized
+        scopes:
+          - openid
+          - profile
+          - groups
+          - email
+        response_types:
+          - code
+        grant_types:
+          - authorization_code
```
Comment (james-d-elliott) [inline docs/content/en/integration/openid-connect/powerdns/index.md:93]:
```suggestion
        authorization_policy: 'two_factor'
        redirect_uris:
          - 'https://powerdns.example.com/oidc/authorized'
        scopes:
          - ''openid'
          - 'profile'
          - 'groups'
          - 'email'
        response_types:
          - 'code'
        grant_types:
          - 'authorization_code'
```

File: docs/content/en/integration/openid-connect/powerdns/index.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+which will operate with the above example:
+
+```yaml
+identity_providers:
+  oidc:
+    ## The other portions of the mandatory OpenID Connect 1.0 configuration go here.
+    ## See: https://www.authelia.com/c/oidc
+    clients:
+      - id: powerdns
+        description: PowerDNS
```
Comment (james-d-elliott) [inline docs/content/en/integration/openid-connect/powerdns/index.md:79]:
```suggestion
      - id: 'powerdns'
        description: 'PowerDNS'
```

**Review by james-d-elliott — CHANGES_REQUESTED** [review]
Few changes otherwise LGTM. Wouldn't happen to know what client auth mechanism they use would you?

File: docs/content/en/integration/openid-connect/powerdns/index.md (hunk 2 above)
Comment (Kitof) [reply docs/content/en/integration/openid-connect/powerdns/index.md:93]: Be careful : double quote on ''openid'

File: docs/content/en/integration/openid-connect/powerdns/index.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+toc: true
+community: true
+---
+
+## Tested Versions
+
+* [Authelia]
+  * [v4.38.0](https://github.com/authelia/authelia/releases/tag/v4.38.0)
+* [PowerDNS Admin]
+  * [v0.4.1]([https://github.com/PowerDNS-Admin/PowerDNS-Admin](https://github.com/PowerDNS-Admin/PowerDNS-Admin/releases/tag/v0.4.1))
```
Comment (james-d-elliott) [inline docs/content/en/integration/openid-connect/powerdns/index.md:21]:
```suggestion
  * [v0.4.1](https://github.com/PowerDNS-Admin/PowerDNS-Admin/releases/tag/v0.4.1)
```

**Review by james-d-elliott — APPROVED** [review]
LGTM just making this one change.

**Pull request merged**